yoti_python_sdk/attribute.py

```diff
@@ -0,0 +1,18 @@
+class attribute:
+    name = ""
+    value = ""
+    anchors = {}
+
+    def __init__(self, name, value, anchors):

```

Comment:
Are we using 'anchors' already instead of 'sources' and 'verifiers'?

Reply:
We have all 3 available: `.anchors`, `.sources` and `.verifiers` are present, with the latter two just being filters on the former.